test/test.js

```diff
@@ -1,6 +1,6 @@
 const test = require('tape');
 
-const app = Container.loadAndInstantiate("dist/bundle.json")
+const app = Container.loadAndInstantiate("dist/app-spec-rust.hcpkg")
```

Comment:
If you switch `build_and_test.sh` like I suggest, this needs to be `dist/bundle.json`, not `dist/app-spec-rust.hcpkg`